File: Python/8. 뒤집은 소수/AA.py
import sys
sys.stdin=open('input.txt', 'rt')

n=int(input())
arr=list(map(int, input().split()))

temp=''
nums=[0]*100

def isPrime(a):
    flag=True
    for i in range(2, len(arr)+1):
        if a%i==0:
            flag=False
            break
    if flag:
        print(a, end=' ')

for i in range(len(arr)):
    num=arr[i]
    temp=str(num)
    temp2=''
    for j in range(len(temp)):
        temp2.__add__(temp[len(temp)-1:])
    reverse=int(temp2)

    isPrime(reverse)

File: Javascript/섹션2. 1-2차원 배열 탐색/1. 큰 수 출력하기/AA.js
function solution(num, arr){
    let answer=[]
    
    for(let i=0;i<arr.length;i++){
        console.log(i)
        if(i==0) {
            answer.push(arr[i])
            continue
        }
        console.log(arr[i], arr[i-1])
        console.log(arr[i] > arr[i-1])
        
        if(arr[i+100]>arr[i-1]){
            answer.push(arr[i])
            console.log(answer)
        }
    }

    return answer
}
let str="7 3 9 5 6 12"
arr=str.split(' ')
console.log(solution(6, arr))